TestProject/ClientApp/src/components/Layout.js

```diff
@@ -27,14 +30,30 @@ function Layout() {
             });
     }, [])
 
+    async function handleGetUpdatedRating(postId) {
```

Comment:
`handle` is superfluous here

Reply:
and this shouldn't be placed here, you can create a separate service for such a requests